### ISSUE TITLE
ubutu 20.4 e5-2698  err

### ISSUE THREAD

**Issue by xieshunwan** [madMAx43v3r/chia-plotter#659]
Multi-threaded pipelined Chia k32 plotter - 78ee372
Final Directory: /mnt/a/
Number of Plots: 1
Crafting plot 1 out of 1
Process ID: 132235
Number of Threads: 16
Number of Buckets P1:    2^9 (512)
Number of Buckets P3+P4: 2^9 (512)
Pool Public Key:   a4e6884a8714afee0eee40f124ad2b98003446e2d646c56f7d7c48ef54385d96d0f76478b43f9bf7550c6b7105632b5b
Farmer Public Key: aecc3d3ef7626bcec537c3ba0e79b7b6d6fc30d2ea0fc6108084c955a4f1835dcc5b16b56c8ab808798eab2c13d45178
Working Directory:   /mnt/tmp1/
Working Directory 2: /mnt/ram/
Plot Name: plot-k32-2021-06-24-16-15-d1f6702d078df6624165eadb4f923a58975edd77e51e6c5a2e5f315c7a5a4e24
[P1] Table 1 took 17.2663 sec
[P1] Table 2 took 122.253 sec, found 4295081799 matches
[P1] Table 3 took 132.184 sec, found 4295172491 matches
[P1] Table 4 took 216.219 sec, found 4295187086 matches
[P1] Table 5 took 179.827 sec, found 4295132314 matches
[P1] Table 6 took 173.047 sec, found 4295045696 matches
[P1] Table 7 took 116.641 sec, found 4295055910 matches
Phase 1 took 957.469 sec
[P2] max_table_size = 4295187086
[P2] Table 7 scan took 10.8949 sec
[P2] Table 7 rewrite took 32.8274 sec, dropped 0 entries (0 %)
terminate called after throwing an instance of 'std::runtime_error'
  what():  fopen() failed

**Comment by andymarden** [comment]
Yes I have started to get similar errors - all of sudden a couple of days ago - it will do a few plot a and the something like this happens. MI have had various similar errors - the last one is this:

Phase 2 took 959.948 sec
Wrote plot header with 268 bytes
[P3-1] Table 2 took 76.3001 sec, wrote 3429405955 right entries
terminate called after throwing an instance of 'std::runtime_error'
  what():  thread failed with: thread failed with: small_delta >= 256 (18488492036)
Aborted (core dumped)

**Comment by andymarden** [comment]
I am on up to date Ubuntu 21.04

**Comment by lemur73** [comment]
> Yes I have started to get similar errors - all of sudden a couple of days ago - it will do a few plot a and the something like this happens. MI have had various similar errors - the last one is this:
> 
> Phase 2 took 959.948 sec
> Wrote plot header with 268 bytes
> [P3-1] Table 2 took 76.3001 sec, wrote 3429405955 right entries
> terminate called after throwing an instance of 'std::runtime_error'
> what(): thread failed with: thread failed with: small_delta >= 256 (18488492036)
> Aborted (core dumped)

IMHO, this one is pretty different.
I would suggest you to run RAM test.

**Comment by lemur73** [comment]
> terminate called after throwing an instance of 'std::runtime_error'
> what(): fopen() failed

Please, try to increase opened files limit.

**Comment by andymarden** [comment]
My issue seems fine now

**Comment by matiaso** [comment]
So what solved this issue ?
